### PR TITLE
Upgrade Decidim to 0.25

### PR DESCRIPTION
**Pull request by LauraJaimeC**
#### :tophat: What? Why?
Upgrade Decidim to 0.25

- Upgrade all modules
- Upgrade gems
- Remove espais-estables module and move locales to app
- Review overrides and decorators

TODO: 

- [x] https://github.com/gencat/decidim-module-department_admin/pull/52
- [x] https://github.com/gencat/decidim-verifications-members_picker/pull/4

### CODE REVIEW

**Review by tramuntanal — CHANGES_REQUESTED** [review]
Very good job here @LauraJaimeC !

This is a first check, we'll QA and see how this complex upgrade works. Please apply the following sugestions when you have the time.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -10,9 +10,10 @@ on:
 
 env:
   RUBY_VERSION: 2.7.2
+  NODE_VERSION: 16.9.1
 
 jobs:
-  test-app:
+  test-report:
```
[the remark this answers is not in the document]
Comment (LauraJaimeC) [reply .github/workflows/test.yml:16]: 😅

File: decidim-process-extended/app/decorators/decidim/participatory_processes/admin/copy_participatory_process_decorator.rb [hunk 2]
```diff
@@ -10,8 +10,6 @@ def copy_participatory_process
       hashtag: @participatory_process.hashtag,
       description: @participatory_process.description,
       short_description: @participatory_process.short_description,
-      hero_image: @participatory_process.hero_image,
-      banner_image: @participatory_process.banner_image,
```
[the remark this answers is not in the document]
Comment (LauraJaimeC) [reply decidim-process-extended/app/decorators/decidim/participatory_processes/admin/copy_participatory_process_decorator.rb:14]: This change coming from Decidim: 

https://github.com/CodiTramuntana/decidim/blob/release/0.25-stable/decidim-participatory_processes/app/commands/decidim/participatory_processes/admin/copy_participatory_process.rb

File: decidim-process-extended/app/decorators/decidim/participatory_processes/admin/update_participatory_process_decorator.rb [hunk 3]
```diff
@@ -8,10 +8,6 @@ def attributes
       weight: form.weight,
       slug: form.slug,
       hashtag: form.hashtag,
-      hero_image: form.hero_image,
-      remove_hero_image: form.remove_hero_image,
-      banner_image: form.banner_image,
-      remove_banner_image: form.remove_banner_image,
```
[the remark this answers is not in the document]
Comment (LauraJaimeC) [reply decidim-process-extended/app/decorators/decidim/participatory_processes/admin/update_participatory_process_decorator.rb:14]: This change also comes from Decidim but some code is missing here...i'll change it..

File: db/migrate/20190114123610_move_organization_fields_to_hero_content_block.decidim.rb [hunk 4]
```diff
@@ -4,7 +4,7 @@
 
 class MoveOrganizationFieldsToHeroContentBlock < ActiveRecord::Migration[5.2]
   class ::Decidim::Organization < Decidim::ApplicationRecord
-    mount_uploader :homepage_image, ::Decidim::HomepageImageUploader
+    # mount_uploader :homepage_image, ::Decidim::HomepageImageUploader
```
[the remark this answers is not in the document]
Comment (LauraJaimeC) [reply db/migrate/20190114123610_move_organization_fields_to_hero_content_block.decidim.rb]: The reason:
imagen

File: bin/setup [hunk 5]
```diff
@@ -20,6 +20,9 @@ chdir APP_ROOT do
   system! "gem install bundler --conservative"
   system("bundle check") || system!("bundle install")
 
+  # Install JavaScript dependencies
+  system! "bin/yarn"
```
[the remark this answers is not in the document]
Comment (tramuntanal) [reply bin/setup]: this is still pending

File: db/migrate/20190114123610_move_organization_fields_to_hero_content_block.decidim.rb (hunk 4 above)
[the remark this answers is not in the document]
Comment (tramuntanal) [reply db/migrate/20190114123610_move_organization_fields_to_hero_content_block.decidim.rb]: ok, then..

File: decidim-process-extended/app/decorators/decidim/participatory_processes/admin/copy_participatory_process_decorator.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (tramuntanal) [reply decidim-process-extended/app/decorators/decidim/participatory_processes/admin/copy_participatory_process_decorator.rb:14]: ok, now I see there's a `copy_participatory_process_attachments` method in the original class

**Review by tramuntanal — APPROVED** [review]
Good job @LauraJaimeC !

**Comment by LauraJaimeC** [comment]
Superseded by https://github.com/gencat/participa/pull/347

**Pull request closed without merging**